File: 3D_08_Łazowy_Hubert/4.5.cpp
#include "funk.h"
#define max 100

using namespace std;

string sprczyw(int tab[max] ,int size){
    for(int i=0;i<size;i++){
        for(int j = i+1;j<size;j++){
            if( tab[i] == tab[j] ){return "Tak";}
        }
    }
    return "Nie";
}
int main(){
    srand(time(NULL));
    bool re = true;
    do{
        int size = input("Jaki ma byc rozmiar tablicy: ");        
        int tab[size];
        cout<<"podajemy ? t or n\n";
        char ans = getin();
        if(ans == 't'){
            czytajt(tab, size);
        }
        else{
            lost(tab, size);
        }
        piszt(tab, size);
        cout<<"\nCzy sie powtarza: "<<sprczyw(tab,size);
        jeszcze(re);
    }while(re==true);
    return 0;
}

File: 3D_08_Łazowy_Hubert/4.5.py
from funk import *
max = 5

def sprczyw(tab, size):
    conv = set(tab)
    if len(conv) < size:
        return "Tak"
    return "Nie"

def main():
    re = True
    while re == True:
        size = inputh("Jaki ma być rozmiar tablicy:")
        tab = [0,]*size
        print("podajemy ? (T/N)")
        ans = getin()
        if ans == b't':
            czytajt(tab,size)
        else:
            lost(tab,size)
        piszt(tab,size)
        print(f"czy sie powetarza:{sprczyw(tab,size)}")
        re = jeszcze()

if __name__ == "__main__":
    main()

